sphinx_doc/getting_started/index.rst

```diff
@@ -3,5 +3,129 @@
 Getting Started
 ###############
 
-This website is under construction. Please visit the `project repo`_ in the
-meantime.
+This document will take you through the process of installing the |PN| binaries
+and navigating a simulated Turtlebot 3 in the Gazebo simulator. We'll use the
+|Distro| version of ROS 2, since it is the latest stable version at the time
+this was written. The instructions are written primarily for Ubuntu 18, using
+the standard installation options.
+
+.. note::
+
+  See the :ref:`howtos` for other situations such as building from source or
+  working with other types of robots.
+
+.. warning::
+
+  This is a simplified version of the Turtlebot 3 instructions. We highly
+  recommend you follow the `official Turtlebot 3 manual`_ if you intend to
+  continue working with this robot beyond the minimal example provided here.
+
+Installation
+************
+
+1. Install the `ROS 2 binary packages`_ as described in the official docs
+2. Install the |PN| packages using your operating system's package manager. For
+   Ubuntu 18, do this:
+
+   .. code-block:: bash
+
+      sudo apt install ros2-dashing-navigation2
+      sudo apt install ros2-dashing-nav2-bringup
+
+3. Install the Turtlebot 3 packages. Again, for Ubuntu 18, it looks like this:
+
+   .. code-block:: bash
+
+      sudo apt install ros2-dashing-turtlebot3*
+
+Running the Example
```

Comment:
Is there a way to load the markdown file with these same instructions to be viewed? I would ideally not like to have multiple of the same instructions we need to keep in sync.

If we're going the sphinx route, should we just have this be the docs and remove the markdown if we cant link them?

Reply:
> If we're going the sphinx route, should we just have this be the docs and remove the markdown if we cant link them?

Yeah. That was my thinking. Sphinx would be the main set of docs, and the markdown we have would get transitioned to there.

We can put a link in the sphinx docs to point back to github markdown for stuff we haven't transitioned or don't intend to.

Reply:
I wasn't sure if you could "render" the markdown file as the webpage (similar to loading a picture or something from the md file). If not, I'd be OK with merging this and then just deleting the md file if they match up.

We  should have the live master sphinx page in the main project readme